core/modules/user/user.module

```diff
@@ -498,20 +498,20 @@ function user_is_blocked($name) {
 /**
  * Checks if a user has a role.
  *
- * @param int $rid
- *   A role ID.
+ * @param string $role_name
+ *   A string representing the role name.
```

Comment:
```suggestion
 *   The machine name of the role.
```

Reply:
I copied that text directly from a function in the same file. I prefer to keep them consistent.